modules/RTC/TraceablePeerConnection.js

```diff
@@ -2053,6 +2061,40 @@ TraceablePeerConnection.prototype.setVideoTransferActive = function(active) {
     return changed;
 };
 
+/**
+ * Sends DTMF tones if possible.
+ *
+ * @param {string} tones - The DTMF tones string as defined by {@code RTCDTMFSender.insertDTMF}, 'tones' argument.
+ * @param {number} duration - The amount of time in milliseconds that each DTMF should last. It's 200ms by default.
+ * @param {number} interToneGap - The length of time in miliseconds to wait between tones. It's 200ms by default.
+ *
+ * @returns {void}
+ */
+TraceablePeerConnection.prototype.sendTones = function(tones, duration = 200, interToneGap = 200) {
+    if (!this._dtmfSender) {
+        if (this.peerconnection.getSenders) {
+            this._dtmfSender = this.peerconnection.getSenders().map(s => s.dtmf)[0];
+            this._dtmfSender && logger.info(`${this} initialized DTMFSender using getSenders`);
+        }
+
+        if (!this._dtmfSender) {
+            const localAudioTrack = Array.from(this.localTracks.values()).find(t => t.isAudioTrack());
+
+            this._dtmfSender
+                = this.peerconnection.createDTMFSender
+                && localAudioTrack
```

Comment:
_dtmfSender could become a boolean. Not sure if that matters.

Reply:
I was thinking it doesn't, because still falsy, but not 100% accurate.